platform-includes/logs/setup/unreal.mdx

```diff
@@ -0,0 +1,132 @@
+To enable logging in your Unreal Engine project, you need to configure the Sentry SDK with structured logging enabled.
+
+### Project Settings Configuration
+
+1. Open your project settings: **Project Settings > Plugins > Sentry**
+2. Check the **Enable Structured Logging** option
+
+### Programmatic Configuration
+
+Alternatively, you can enable logging programmatically when initializing the SDK:
+
+```cpp
+#include "SentrySubsystem.h"
+
+void ConfigureSentryWithLogs()
+{
+    USentrySubsystem* SentrySubsystem = GEngine->GetEngineSubsystem<USentrySubsystem>();
+
+    // Create settings with logging enabled
+    USentrySettings* Settings = NewObject<USentrySettings>();
+    Settings->EnableStructuredLogging = true;
+
+    // Initialize with custom settings
+    SentrySubsystem->InitializeWithSettings(Settings);
+}
+```
+
+### Advanced Configuration Options
+
+#### Automatic Unreal Engine Log Forwarding
+
+You can configure the SDK to automatically forward Unreal Engine's native `UE_LOG` calls to Sentry based on the enabled severity levels:
+
+```cpp
+// Configure automatic log forwarding programmatically
+USentrySettings* Settings = NewObject<USentrySettings>();
+Settings->EnableStructuredLogging = true;
+
+// Enable specific severity levels for UE_LOG forwarding
+Settings->EnableDebugLogs = false;
+Settings->EnableInfoLogs = true;
+Settings->EnableWarningLogs = true;
+Settings->EnableErrorLogs = true;
+Settings->EnableFatalLogs = true;
+
+Settings->SendLogsAsBreadcrumbs = false; // Send as structured logs instead of breadcrumbs
```

Comment:
There is no such setting in Unreal SDK, probably `bSendBreadcrumbsWithStructuredLogging` was meant?